docs/index.md

```diff
@@ -0,0 +1,93 @@
+# What is Struct Frame
+
+Struct Frame is a code generation framework that converts Protocol Buffer (.proto) files into serialization code for C, C++, TypeScript, Python, and GraphQL. It provides message framing and parsing for structured communication over serial links, sockets, or any byte stream.
```

Comment:
Struct frame is a packeting, framing, and parsing framework that is cross platform. That is the main thing purpose. It does this by using .proto files and generating code from that.